applications/services/cli/cli_vcp.c

```diff
@@ -103,6 +121,15 @@ static int32_t vcp_worker(void* context) {
     FURI_LOG_D(TAG, "Start");
     vcp->running = true;
 
+    // GUI icon
+    vcp->desktop = furi_record_open(RECORD_DESKTOP);
```

Comment:
![image](https://github.com/user-attachments/assets/f37e6687-951c-4934-86e1-fe0f10563460)

I don't think that we need one more icon in status bar.

Reply:
I think it's useful to have an icon that tells the user why their device is not autolocking. Maybe the solution is to hide the CLI icon if an RPC session is active; it's kinda redundant.

Reply:
Resolved with the UI/UX team.